src/main/java/io/redisearch/aggregation/AggregationRequest.java

```diff
@@ -96,6 +107,15 @@ public AggregationRequest groupBy(Group group) {
         return this;
     }
     
+    public AggregationRequest filter(String expression) {
```

Comment:
Isn't the following enough? 
```
    public AggregationRequest filter(String expression) {
  	filterQuery = expression;
    	return this;
    }
```

Reply:
RediSearch allows multiple FILTER expressions. If we directly it assign it to filterQuery it will overwrite the previous filter expression.To support developers adding multiple filter expression kept this logic.

Reply:
Right, but it can be either && or || and you just provided a way to add `and` clause, so I think it's better to avoid confusion and just let the user send a full filter string.

Reply:
When multiple FILTER expressions are used they are "&&" in RediSearch.